### DISCUSSION TITLE
What are the requirements to get it to work on MacOS

### DISCUSSION THREAD

**Discussion by Zambrella** [alexmercerind/dart_vlc#358] [Q&A]
Hi,

In some parts of the documentation it says that this package support MacOS but when I call `DartVLC.initialize();` I get an error of `Failed to load dynamic library`.

I have VLC installed, are there any other requirements?

Thanks.

**Comment by Spokplacenta** [comment]
I think dart_vlc use VLCKit.
Since version 0.3 MacOS is no more supported, I don't know why but it's au problem for me. :/
